conjure_oxide/tests/integration/minion_constraints/div_undefzero-03-nested/input.eprime

```diff
@@ -0,0 +1,7 @@
+language ESSENCE' 1.0
+
+find x : int(5..20)
+find y : int(0..5)
+find z : int(0..6)
+
+such that x / (y/z) = 10
```

Comment:
thanks. this is coming bit by bit but I think two more separate tests would be good to be a bit more comprehensive here. hopefully it is now trivial to add new test cases.

- same vars, but constraint `x/(y/z) != 10`
- same vars, but constraint `!(x/(y/z) = 10)`

Reply:
The last one found a bug; thanks for that suggestion!

This might also fix my (unpublished) attempts at mixing nested sums and divisions...